script.lua

```diff
@@ -7733,16 +7733,93 @@ function onObjectDestroy(obj)
         end
     end
 end
-function getCardImageURL(card)
-    local cardData = card.getData()
-    if cardData.CustomDeck then
-        for _, data in pairs(cardData.CustomDeck) do
-            if data.NumWidth == 1 and data.NumHeight == 1 then
-                return data.FaceURL
+function getReminderLocation(params)
+    local obj = params.obj
+    if obj == nil then
+        return nil
+    end
+
+    -- Get it from the script state if it's there
+    local state = {}
+    if obj.script_state ~= "" then
+        state = JSON.decode(obj.script_state)
+    end
+    if state.reminder ~= nil then
+        return state.reminder
+    end
+
+    -- Otherwise, use a sensible default
```

Comment:
are these sensible defaults based on what we used to use?

Reply:
Yeah, they're based on what the mod already uses for cards, and for spirits without "Lower Spirit Image".

That said, they're "based on" them in the sense that I simply lined things up as best as I could by eye, rather than performing the necessary calculations to make it *perfect* (due to the aforementioned fact about the "centres" of the holes in the masks making those calculations difficult). Still, should be close enough that you can't tell the difference.

Reply:
and then since the image mask is different shape, the numbers aren't going to be 1-to-1